cfme/tests/v2v/test_post_migrations.py

```diff
@@ -1,191 +1,100 @@
-"""Test contains all post migration usecases"""
+"""Tests to validate post-migrations usecases"""
 import re
 
 import fauxfactory
 import pytest
 
 from cfme.base.login import BaseLoggedInPage
-from cfme.fixtures.provider import rhel7_minimal
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.fixtures.v2v_fixtures import get_migrated_vm
 from cfme.infrastructure.provider.rhevm import RHEVMProvider
 from cfme.infrastructure.provider.virtualcenter import VMwareProvider
 from cfme.markers.env_markers.provider import ONE_PER_TYPE
 from cfme.markers.env_markers.provider import ONE_PER_VERSION
 from cfme.utils.appliance.implementations.ui import navigate_to
-from cfme.utils.appliance.implementations.ui import navigator
-from cfme.utils.log import logger
 from cfme.utils.wait import TimedOutError
-from cfme.utils.wait import wait_for
-
 
 pytestmark = [
     pytest.mark.provider(
-        classes=[RHEVMProvider],
+        classes=[RHEVMProvider, OpenStackProvider],
         selector=ONE_PER_VERSION,
-        required_flags=['v2v'],
-        scope="module"
+        required_flags=["v2v"],
+        scope="module",
     ),
     pytest.mark.provider(
         classes=[VMwareProvider],
         selector=ONE_PER_TYPE,
-        fixture_name='source_provider',
-        required_flags=['v2v'],
-        scope="module"
-    )
+        required_flags=["v2v"],
+        fixture_name="source_provider",
+        scope="module",
+    ),
+    pytest.mark.usefixtures("v2v_provider_setup")
 ]
 
 
-def get_migrated_vm_obj(src_vm_obj, target_provider):
-    """Returns the migrated_vm obj from target_provider."""
-    collection = target_provider.appliance.provider_based_collection(target_provider)
-    migrated_vm = collection.instantiate(src_vm_obj.name, target_provider)
-    return migrated_vm
-
-
-@pytest.mark.parametrize(
-    "form_data_vm_obj_single_datastore", [["nfs", "nfs", rhel7_minimal]], indirect=True
-)
-def test_migration_policy_tag(request, appliance, v2v_providers, host_creds, conversion_tags,
-                              form_data_vm_obj_single_datastore, soft_assert):
-    """Test policy to prevent source VM from starting if migration is complete
+@pytest.mark.tier(1)
```

Comment:
**Question** Does this not have to be parametrized over the different datastores anymore?

Reply:
@jawatts, as I see this is different test.

Reply:
@Yadnyawalkya, please confirm that you have merged two tests into one.
btw, please make sure your changes will be correctly tracked in polarion.

Reply:
You both are correct here. :100:

> **Question** Does this not have to be parametrized over the different datastores anymore?

@jawatts : we are not using datastore anymore `mapping_data_vm_obj_mini` will take care of that.
 
> @jawatts, as I see this is different test.

@izapolsk : right, I added two tests into one.